### ISSUE TITLE
Ping you when your name is mentioned

### ISSUE THREAD

**Issue by Aces-and-Jacks** [mrbuilder1961/WheresMyChatHistory#206]
**Is your feature request related to a problem? Please describe.**
Tons of messages, so I can't see if I'm mentioned

**Describe the solution you'd like**
A ping when your name is mentioned. 

**Describe alternatives you've considered**
Downloading a different mod.

**Additional context**
 Not sure if this would be ok or not to add. If it isn't that's fine then. I didn't see anyone else mention this when I searched.

**Comment by JajaGamer** [comment]
yes please

**Comment by mrbuilder1961** [comment]
at least for right now i'd recommend just using [Chat Notify](https://modrinth.com/mod/chatnotify) as it seems to have a lot of features available. once i have some time to play around with it i might add it to the list of suggested mods, but it seems redundant to re-implement a whole set of features when a perfectly good and complex mod already exists to serve that purpose

**Comment by Aces-and-Jacks** [comment]
Yeah, that's what I'm using currently. And it works well with this mod. Should I close this then? Or keep it up

**Comment by mrbuilder1961** [comment]
yeah for now this is not planned. thanks for the suggestion though
